Client/Frontend/Strings.swift

```diff
@@ -405,6 +405,15 @@ extension String {
                 comment: "This value is used as the toast message for the saving success alert in the remember credit card page")
         }
 
+        // Select Credit Card
+        public struct SelectCreditCard {
+            public static let MainTitle = MZLocalizedString(
+                key: "CreditCard.SelectCreditCard.MainTitle.v116",
+                tableName: "UpdateCard",
```

Comment:
I think we want the tableName to be the same as the main struct? At least that's what we have on other strings. 🤔
```suggestion
                tableName: "SelectCreditCard",
```

Reply:
Good find 👍🏼